### ISSUE TITLE
some  problem

### ISSUE THREAD

**Issue by myw8** [bamos/dcgan-completion.tensorflow#13]
Sorry to trouble you again,I use 64*64 grayscale images of Chinese characters  and  trying to repair Chinese characters.But  the completed  and  hats_imgs  seem to  no change
. When training  after a period of training   the g_loss  is a lot of  bigger than  d_loss,   when  I use your method to repair, the loss  is very high,I  have  tried  to change learning rate lam momentum .the loss is  still very high  about  7500，can you help me 。I hope you can understand what I mean, my English is not  good.

**Comment by myw8** [comment]
![aa](https://cloud.githubusercontent.com/assets/17568519/21227566/920c73b4-c315-11e6-83f1-759d5a86b1d4.png)
when  training  the  d_loss and  g_loss,are there any problems with them?

**Comment by myw8** [comment]
![dd](https://cloud.githubusercontent.com/assets/17568519/21227967/015fbc34-c317-11e6-8fbc-1b67f61ade30.png)
My training data set

**Comment by myw8** [comment]
![loss](https://cloud.githubusercontent.com/assets/17568519/21228197/d4962426-c317-11e6-9532-fa4ab60eff7a.png)
![gg1](https://cloud.githubusercontent.com/assets/17568519/21228346/63bfbf18-c318-11e6-9f5a-bd6c0fcd028f.png)
![dd1](https://cloud.githubusercontent.com/assets/17568519/21228373/7cdf747a-c318-11e6-8e11-5235d5be75d9.png)

the  loss  hat_imgs  completed

**Comment by bamos** [comment]
Hi @myw8 - these results could be from a poor generative model or from poor optimization over the latent space. I think we should use ADAM instead of GD to optimize over the latent space. If you implement this please send in a PR and link to issue #9.

-Brandon.

**Comment by myw8** [comment]
I  set  lam=0  and  found  the complete_ loss(contextual_loss) have  no change.  Is  it  because  a poor generative model?  should I build a single channel  generative  model?How  to  do it,Can you recommend some information，thank you very much!

**Comment by myw8** [comment]
I  set  lam=0  and  found  the complete_ loss(contextual_loss) have  no change.  Is  it  because  a poor generative model?  should I build a single channel  generative  model?How  to  do it,Can you recommend some information，thank you very much!
